webpack.config.devtools.js

```diff
@@ -28,6 +27,8 @@ module.exports = webpackConfig => {
     webpackConfig.output.path = path.join(__dirname, "firefox/devtools/client/debugger/new");
   }
 
+  webpackConfig.resolve.alias["public/js/utils/networkRequest"] = "public/js/utils/privilegedNetworkRequest";
```

Comment:
I wonder if it makes sense to move `privilegedNetworkRequest` to m-c under `devtools/shared`. 

* this module will not be used locally 
* this might be more consistent with other shim modules like Prefs where the panel implementation lives in m-c